src/api/services/MessagesService.js

```diff
@@ -1,15 +1,50 @@
 import axios from '../http';
 
 const MessagesService = {
-  fetchMessages: async ({ pageNumber }) => {
-    const response = await axios.get(
-      `/messages?offset=${pageNumber - 1}&pagesize=15`
-    );
-    return response.data;
+  fetchMessages: async (pageNumber) => {
+    const url = `/messages?offset=${pageNumber - 1}&pagesize=15`;
+    try {
+      const response = await axios.get(url);
+      return response.data;
```

Comment:
This is more subjective, in general it is a good practice to seperate return statements with a new line 
```suggestion
      const response = await axios.get(url);
      
      return response.data;
```